### PR TITLE
feat(try-catch): updated all the tests to include catches for exceptions

### PR DESCRIPTION
**Pull request by sudoku-lord**
Fixes DXP-287

## Checklist

- [x] Up to date with `main`
- [x] All the tests are passing
  - [x] Prettier
  - [x] Spectral Lint
  - [x] Contract Tests
    - [x] Delete all resources created in contract tests
- [ ] `npm run bundle` outputs nothing suspect
- [ ] `npm run postman` outputs nothing suspect

_i've gone blind_

(the last two are unchecked bc I have to update my own node version)

### CODE REVIEW

**Comment by jho44** [comment]
- got rid of `t.plan` since it wasn't really adding anything to our error msgs
- added `t.assert(false)` in `catch` so ava doesn't cry about having at least 1 assertion per test
- got rid of the `prismError.message` parts -- I didn't find them as informative as the autogenerated exception messages
- `console.error(prismError)` rather than `console.error(JSON.stringify...)` b/c the latter was just printing `{}`

**Comment by sudoku-lord** [comment]
FINALLY found the best solution to the `{}` pretty-printing issue.

File: tests/addresses_test.js [hunk 1]
```diff
@@ -13,24 +13,40 @@ const prism = new Prism(specFile, lobUri, process.env.LOB_API_TEST_TOKEN);
...
+      );
+      t.assert(response.status === 200);
+      return response.data;
+    } catch (prismError) {
+      if (Object.keys(prismError).length > 0) {
+        t.fail(JSON.stringify(prismError, null, 2));
+      } else {
+        t.fail(prismError.toString());
+      }
+      return prismError;
```
Comment (jho44) [inline tests/addresses_test.js:48]: if the flow comes into this branch, it'd never make it to this `return` right? cuz the `t.fail` would stop the whole test?

**Review by jho44 — APPROVED** [review]
Beyond that one comment I made, everything seems good. And even if u leave that `return` in, it doesn't change the effect

File: tests/addresses_test.js (hunk 1 above)
Comment (sudoku-lord) [reply tests/addresses_test.js:48]: ah crap the return should be in the else block, thanks for catching that

**Pull request merged**